polyfill/intersection-observer.js

```diff
@@ -254,6 +254,8 @@ IntersectionObserver.prototype._monitorIntersections = function() {
     else {
       addEvent(window, 'resize', this._checkForIntersections, true);
       addEvent(document, 'scroll', this._checkForIntersections, true);
+      addEvent(document, 'transitionend', this._checkForIntersections, true);
```

Comment:
We need to remove the event on unobserve()

Reply:
You are right.